playwright/tests/api.spec.ts

```diff
@@ -12,3 +12,19 @@ test("API Test - GET Products", async ({ request }) => {
     })
   );
 });
+
+test.only("API Test - GET Product #5", async ({ request }) => {
+  const requestURL5 = "https://fakestoreapi.com/products/5";
+  const response5 = await request.get(requestURL5);
+  const responseBody5 = await response5.json();
+  await expect(response5.ok()).toBeTruthy();
+  await expect(response5.status()).toBe(200);
+  await expect(responseBody5).toMatchObject(
```

Comment:
Great one! You noticed that now we are testing single object response instead of array of objects!